packages/rum-core/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "elastic-apm-js-core",
+  "version": "3.0.1",
+  "description": "Elastic apm core",
+  "license": "MIT",
+  "main": "src/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elastic/apm-agent-js-core.git"
+  },
+  "scripts": {
+    "prepare-patch": "sh ../../dev-utils/prepare-release patch && npm run push-release",
+    "prepare-minor": "sh ../../dev-utils/prepare-release minor && npm run push-release",
+    "prepare-major": "sh ../../dev-utils/prepare-release major && npm run push-release",
+    "push-release": "git push origin master --follow-tags && npm publish",
+    "changelog": "node node_modules/conventional-changelog-cli/cli.js  -p angular -i CHANGELOG.md -s",
+    "karma": "karma start",
+    "karma-bench": "karma start karma.bench.conf.js",
+    "karma-coverage": "karma start --coverage",
+    "runUnitTests": "node ../../dev-utils/run-script.js runUnitTests true packages/rum-core",
+    "test": "npm run runUnitTests",
+    "lint": "standard",
+    "format": "prettier-standard 'dev-utils/**/*.js' src/**/*.js test/**/*.js"
```

Comment:
Its better to remove prettier-standard and follow the eslint configs that we had for js-base.. its much better and works for licensing header checks.. 

We can do this also as separate PR if you feel its too much.

Reply:
The lint script is on the root package.json, I've removed these scripts now!